receive_test.go

```diff
@@ -98,42 +99,6 @@ func TestCopyWithSubDir(t *testing.T) {
 	assert.Equal(t, "data1", string(dt))
 }
 
-func TestCopyDirectoryTimestamps(t *testing.T) {
```

Comment:
Also looks removed

Reply:
I can't really explain what malfunction I had that made me decide to remove this. Added it back, as well as the mod times.